flow-typed/declarations.js.flow

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-undef */
+
+declare type LdClientWrapperType = {
+  on: (string, (FlagValueType) => (void)) => void,
+  onReady: (() => void) => void,
+  variation: (string, boolean) => FlagValueType
+};
+
+declare type FeatureFlagType = {
+  flagKey: string,
+  renderFeatureCallback: (FlagValueType) => ?React$Element<any>,
+  renderDefaultCallback?: () => ?React$Element<any>,
+  initialRenderCallback?: () => ?React$Element<any>
+};
+
+declare type ConfigType = {
```

Comment:
These all need to be wrapped inside:
```
declare module 'react-launch-darkly' {
```

Otherwise, they'll be declared as globals, which silently clobber existing types, like UserType.

Reply:
Example:
https://github.com/flowtype/flow-typed/blob/master/definitions/npm/axios_v0.17.x/flow_v0.25.x-/axios_v0.17.x.js